buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/JUnitXmlTestEventsGenerator.groovy

```diff
@@ -51,41 +54,36 @@ class JUnitXmlTestEventsGenerator {
         this.testListenerBroadcast = testListenerBroadcast
     }
 
-    void processXmlFile(File resultsFile, Object build) {
```

Comment:
Can we now use `@CompileStatic` on the whole class?

Reply:
So only `processTestSuite`?

Reply:
Sure.